src/quantize.rs

```diff
@@ -15,12 +15,8 @@ use std::mem;
 
 pub fn get_log_tx_scale(tx_size: TxSize) -> i32 {
   let num_pixels = tx_size.area();
-  let mut shift_bits = 0;
 
-  if num_pixels > 256 { shift_bits += 1; };
-  if num_pixels > 1024 { shift_bits += 1; };
-
-  shift_bits
+  (num_pixels > 256) as i32 + (num_pixels > 1024) as i32
```

Comment:
I think I tried exactly the same code in the beginning, borrowing from libaom, then rustc cannot pass it, since it does not do automatic type conversion (i.e. casting) from boolean to integer. :) Hence the CI raised red flag!

Reply:
Oh you did casting, "as i32", sorry missed it-- *)*

Reply:
When casted, does rustc generate similar code as C? I,e, ture/false --> 1/0?

Reply:
You can use `as` or `i32::from()` like [this](https://play.rust-lang.org/?version=stable&mode=debug&edition=2015&gist=e5986e05f5837950a0c167f6ba5f0ade)

Reply:
and yes, boolean -> signed maps to 0/1 and not 0/-1 ^^

Reply:
Ok- thanks for confirm!

Reply:
@lu-zero , And thanks for example code for type conversion above!

Reply:
When the type may change `i32::from()` is suggested by clippy since would make refactoring easier. I consider that debatable.

Reply:
@lu-zero I think using `i32::from()` is reasonable.